scene/resources/animation.h

```diff
@@ -401,7 +401,7 @@ class Animation : public Resource {
 	void track_set_key_transition(int p_track, int p_key_idx, real_t p_transition);
 	void track_set_key_value(int p_track, int p_key_idx, const Variant &p_value);
 	void track_set_key_time(int p_track, int p_key_idx, double p_time);
-	int track_find_key(int p_track, double p_time, FindMode p_find_mode = FIND_MODE_NEAREST) const;
+	int track_find_key(int p_track, double p_time, FindMode p_find_mode = FIND_MODE_NEAREST, bool p_backward = false) const;
```

Comment:
You also need to update the bind for this, on line 3856 in the cpp file, you also need to register a compatibility method for this, will provide an example for how to do this, this makes me wonder if this is the best solution however